contents/academy.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: PostHog Academy
+showTitle: false
+showSidebar: true
+width: md
+noindex: true
+images:
+    - './images/startup-hogs.png'
+---
+
+import { StaticImage } from 'gatsby-plugin-image'
+import { getImage, GatsbyImage } from 'gatsby-plugin-image'
+import CommunityCTA from 'components/CommunityCTA'
+
+<Hero
+    title="PostHog Academy"
+    subtitle="Curated lesson tracks from our engineers and users, to help you get even more from PostHog"
+/>
+
+PostHog Academy helps Engineering and Product teams understand how to make the most of PostHog, and build better products. It's a curated track of tutorials and other lessons which cover common use-cases for particular roles, as well as general advice for all users - and it's constantly expanding!
+
+<p class="my-4 flex space-x-2 items-center justify-center">
+<CallToAction href="/tutorials">Browse all tutorials</CallToAction> 
+<CallToAction href="/questions" type="outline">Ask a question</CallToAction>
+</p>
+
+### PostHog 101
+
+Want a quick summary of the essential information you need to know? Look no further than PostHog 101!
+
+<TutorialsSlider slugs={[
+  "/tutorials/non-technical-guide-to-data",
+  "/tutorials/event-tracking-guide",
+  "/tutorials/regex-basics",
```

Comment:
```suggestion
  "/tutorials/next-steps-after-installing",
```
I think regex is a little bit advanced and a tiny be irrelevant to most new users for "PostHog 101"

Reply:
Is there one you think is better?